### PR TITLE
i18(zh-cn): splashscreen

### PR DESCRIPTION
**Pull request by MojaveHao**
#### What kind of changes does this PR include?

- Translated content

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Splashscreen
+# 初始屏幕
 
-If your webpage could take some time to load, or if you need to run an initialization procedure in Rust before displaying your main window, a splashscreen could improve the loading experience for the user.
+如果你的网页可能需要一些时间来加载，或者你需要在显示主窗口之前在Rust中运行初始化过程，那么启动画面可以改善用户的加载体验。
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
如果你的网页可能需要一些时间来加载，或者你需要在显示主窗口之前在 Rust 中运行初始化过程，那么启动画面可以改善用户的加载体验。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-### Setup
+### 设置
 
-First, create a `splashscreen.html` in your `distDir` that contains the HTML code for a splashscreen. Then, update your `tauri.conf.json` like so:
+首先，在`distDir`中创建一个包含初始屏幕的HTML文件。然后，更新`tauri.conf.json`:
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
首先，在 `distDir` 中创建一个包含初始屏幕的 HTML 文件。然后，更新 `tauri.conf.json`：
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 3]
```diff
@@ -27,27 +27,27 @@ First, create a `splashscreen.html` in your `distDir` that contains the HTML cod
...
 ```
 
-Now, your main window will be hidden and the splashscreen window will show when your app is launched. Next, you'll need a way to close the splashscreen and show the main window when your app is ready. How you do this depends on what you are waiting for before closing the splashscreen.
+现在，您的主窗口将被隐藏，并且启动应用程序时将显示初始屏幕窗口。接下来，你需要一种方法来关闭初始屏幕，并在应用准备就绪时显示主窗口。如何执行此操作取决于在关闭初始屏幕之前要等待的内容。
 
-### Waiting for Webpage
+### 等待网页
 
-If you are waiting for your web code, you'll want to create a `close_splashscreen` [command](command).
+如果你正在等待你的 Web 代码，你需要创建一个`close_splashscreen`[命令](command).
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
如果你正在等待你的 Web 代码，你需要创建一个 `close_splashscreen` [命令](command)。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 4]
```diff
@@ -27,27 +27,27 @@ First, create a `splashscreen.html` in your `distDir` that contains the HTML cod
...
   if let Some(splashscreen) = window.get_window("splashscreen") {
     splashscreen.close().unwrap();
   }
-  // Show main window
+  // 显示主屏幕
   window.get_window("main").unwrap().show().unwrap();
 }
 
-// Register the command:
+// 注册命令:
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
// 注册命令：
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 5]
```diff
@@ -69,18 +69,17 @@ or
 const invoke = window.__TAURI__.invoke
 ```
 
-And finally, add an Event Listener (or just call `invoke()` whenever you want):
+最后，添加一个`invoke()`事件侦听器（或者随时调用它）：
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
最后，添加一个 `invoke()` 事件侦听器（或者随时调用它）：
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 6]
```diff
@@ -69,18 +69,17 @@ or
...
+  // 这将等待窗口加载，但你可以在您想要的任何触发器上运行此函数
   invoke('close_splashscreen')
 })
 ```
 
-### Waiting for Rust
+### 等待 Rust
 
-If you are waiting for Rust code to run, put it in the `setup` function handler so you have access to the `App` instance:
+如果你正在等待 Rust 代码运行，请将其放入`setup`函数处理程序中，以便您可以访问该`App`实例:
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
如果你正在等待 Rust 代码运行，请将其放入 `setup` 函数处理程序中，以便您可以访问该 `App` 实例：
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 7]
```diff
@@ -89,14 +88,14 @@ fn main() {
...
+      // 我们在新任务上执行初始化代码，这样应用程序就不会冻结。
       tauri::async_runtime::spawn(async move {
-        // initialize your app here instead of sleeping :)
+        // 在这里初始化而不是消磨时间
         println!("Initializing...");
         std::thread::sleep(std::time::Duration::from_secs(2));
         println!("Done initializing.");
 
-        // After it's done, close the splashscreen and display the main window
+        // 显示主屏幕
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
        // 显示主窗口
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 8]
```diff
@@ -89,14 +88,14 @@ fn main() {
     .setup(|app| {
       let splashscreen_window = app.get_window("splashscreen").unwrap();
       let main_window = app.get_window("main").unwrap();
-      // we perform the initialization code on a new task so the app doesn't freeze
+      // 我们在新任务上执行初始化代码，这样应用程序就不会冻结。
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
      // 我们在新任务上执行初始化代码，这样应用程序就不会冻结
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md [hunk 9]
```diff
@@ -27,27 +27,27 @@ First, create a `splashscreen.html` in your `distDir` that contains the HTML cod
...
+// 该命令必须是异步的，以便它不会在主线程上运行。
 #[tauri::command]
 async fn close_splashscreen(window: tauri::Window) {
-  // Close splashscreen
+  // 关闭初始屏幕
   if let Some(splashscreen) = window.get_window("splashscreen") {
     splashscreen.close().unwrap();
   }
-  // Show main window
+  // 显示主屏幕
```
Comment (dklassic) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/guides/features/splashscreen.md]:
```suggestion
  // 显示主窗口
```

**Review by dklassic — CHANGES_REQUESTED** [review]
Hi there, thanks for contributing! I think the translation is general nice so just some formatting nitpicks here and there and some consistency update on terminology and I think we're good to go.

**Review by dklassic — APPROVED** [review]

**Review by dreyfus92 — APPROVED** [review]
Thank you peeps! LGTM ✅

**Pull request merged**